.env.dev.example

```diff
@@ -42,6 +43,16 @@ EMAIL_HOST_PASSWORD=emailpassword123
 EMAIL_USE_TLS=true
 EMAIL_USE_SSL=false
 
+# S3 configuration
+# Commented are example values if you use Scaleway instead of AWS
+USE_S3=false
+# AWS_ACCESS_KEY_ID=
+# AWS_SECRET_ACCESS_KEY=
+# AWS_STORAGE_BUCKET_NAME=        # "example-bucket-name"
+# AWS_S3_CUSTOM_DOMAIN=None       # "example-bucket-name.s3.fr-par.scw.cloud"
+# AWS_S3_REGION_NAME=None         # "fr-par"
+# AWS_S3_ENDPOINT_URL=None        # "https://s3.fr-par.scw.cloud"
```

Comment:
If I remember well, here's how it works:

The S3 interface code is handled by the AWS SDK for Python, named Boto3. These `AWS_S3_*` env vars are used by Boto3.

If you use Amazon AWS S3, you only need `AWS_STORAGE_BUCKET_NAME` and `AWS_S3_REGION_NAME`. Boto3 will create an Amazon AWS compatible URLs with these two, for both the endpoint and the server domain.

If you use an alternative S3 compatible service, like DigitalOcean or Scaleway, Boto3 won't know how to make the endpoint URL (for storing) or the domain (for serving), so you'll have to specify both.

`AWS_S3_CUSTOM_DOMAIN` is also useful if you serve whatever is on your bucket through a CDN, like Cloudfront. It can also be used if you want to serve your images through the same domain as your instance (some Nginx config will be useful there)